src/lib/conversion/rotation.h

```diff
@@ -91,8 +91,10 @@ enum Rotation : uint8_t {
 	ROTATION_ROLL_90_PITCH_68_YAW_293 = 38,
 	ROTATION_PITCH_315                = 39,
 	ROTATION_ROLL_90_PITCH_315        = 40,
+	ROTATION_MAX,
 
-	ROTATION_MAX
+	// Rotation Enum reserved for custom rotation using Euler Angles
+	ROTATION_CUSTOM                  = 100
```

Comment:
Nevermind, I didn't realize `100` already made it into mavlink. [MAV_SENSOR_ROTATION_CUSTOM](http://mavlink.io/en/messages/common.html#MAV_SENSOR_ROTATION_CUSTOM)